index.js

```diff
@@ -75,6 +75,12 @@
     if (typeof custom === 'symbol') {
       prototype[custom] = Identity$prototype$show;
     }
+    /* istanbul ignore if */
+    if (typeof Deno !== 'undefined') {
+      if (Deno != null && typeof Deno.customInspect === 'symbol') {
+        prototype[Deno.customInspect] = Identity$prototype$show;
+      }
+    }
```

Comment:
By using nested `if` statements, `/* istanbul ignore if */` permits the second portion of the condition to be untested (as it is within the ignored block).

`Deno != null` handles the pathological case in which `Deno` is defined globally but its value is `null`.

`typeof Deno.customInspect === 'symbol'` is not strictly necessary, but seems sensible.